tests/test_validator.py

```diff
@@ -213,6 +226,44 @@ def test_verify_there_is_example_repetition_in_intents(nlu_data_path: Text):
     assert not validator.verify_example_repetition_in_intents(False)
 
 
+def test_verify_logging_message_for_intent_not_used_in_nlu(
+    caplog, validator_under_test
```

Comment:
can we please add type annotations (same for the others)?

Reply:
Yes; however, if that's what we want, and everywhere, I suggest we should enforce this with a linter.

Reply:
For instance, in this file, the `caplog` that is used to capture log output is not annotated anywhere, the type would even need to be imported into the file first from a private package. I'll do this now.